src/stages/configure.ts

```diff
@@ -1,11 +1,22 @@
-import { getInput, startGroup, endGroup } from "@actions/core";
+import { getInput, startGroup, endGroup, setFailed, info } from "@actions/core";
 import { exec } from "@actions/exec";
 
 export const configure = async () => {
   startGroup("Configure attic");
-  const endpoint = getInput("endpoint");
-  const token = getInput("token");
 
-  await exec("attic", ["login", "--set-default", "ci", endpoint, token]);
+  try {
+    const endpoint = getInput("endpoint");
+    const cache = getInput("cache");
+    const token = getInput("token");
+
+    info("Logging in to attic cache");
+    await exec("attic", ["login", "--set-default", cache, endpoint, token]);
+    // remember existing source paths
+    info("Collecting store paths before build");
+    await exec(`${__dirname}/list-nix-store.sh > /tmp/store-path-pre-build`);
```

Comment:
Can this be implemented in the Node.js script directly?

Reply:
yeah, i just thought this was a lot easier - plus it follows what cachix does which is good enough ™️